app-config.yaml

```diff
@@ -21,8 +21,9 @@ app:
         headerColor1: ${HEADER_DARK_COLOR_1}
         headerColor2: ${HEADER_DARK_COLOR_2}
         navigationIndicatorColor: ${NAV_INDICATOR_DARK_COLOR}
+
 organization:
-  name: My Org
+  name: ${ORGANIZATION_NAME}
```

Comment:
It looks like this introduces new variable, we should document it in https://github.com/janus-idp/backstage-showcase/blob/main/showcase-docs/getting-started.md